codegen-client/src/test/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/protocol/ProtocolTestGeneratorTest.kt

```diff
@@ -235,60 +236,47 @@ class ProtocolTestGeneratorTest {
             override fun supportsCodegenContext(clazz: Class<out CodegenContext>): Boolean =
                 clazz.isAssignableFrom(ClientCodegenContext::class.java)
         }
-        val visitor = CodegenVisitor(
-            pluginContext,
-            codegenDecorator,
-        )
-        visitor.execute()
-        println("file:///$testDir/src/operation.rs")
-        return testDir
+        return clientIntegrationTest(model, additionalDecorators = listOf(codegenDecorator)) { _, _ -> }
```

Comment:
What does the `{ _, _ -> }` at the end mean? Is that a closure that ignores two args and then does nothing? It looks so weird.

Reply:
yeah, normally it accepts a lambda. Let me put in a default arg for that, good idea.